bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -99,7 +102,7 @@ protected static synchronized Stats getStats() {
    * @author sduskis
    *
    */
-  private class InstrumentedChannel extends ManagedChannel {
+  public static class InstrumentedChannel extends ManagedChannel {
```

Comment:
Can we keep InstrumentedChannel private to ChannelPool? or at least package private?